app/src/main/java/com/topjohnwu/magisk/ui/settings/SettingsFragment.kt

```diff
@@ -224,26 +226,26 @@ class SettingsFragment : BasePreferenceFragment() {
     private fun setLocalePreference(lp: ListPreference) {
         lp.isEnabled = false
         availableLocales.map {
-                val names = mutableListOf<String>()
-                val values = mutableListOf<String>()
-
-                names.add(
-                    ResourceMgr.getString(defaultLocale, R.string.system_default)
-                )
-                values.add("")
+            val names = mutableListOf<String>()
+            val values = mutableListOf<String>()
 
-                it.forEach { locale ->
-                    names.add(locale.getDisplayName(locale))
-                    values.add(locale.toLangTag())
-                }
+            names.add(
+                ResourceMgr.getString(defaultLocale, R.string.system_default)
+            )
+            values.add("")
 
-                Pair(names.toTypedArray(), values.toTypedArray())
-            }.subscribeK { (names, values) ->
-                lp.isEnabled = true
-                lp.entries = names
-                lp.entryValues = values
-                lp.summary = currentLocale.getDisplayName(currentLocale)
+            it.forEach { locale ->
+                names.add(locale.getDisplayName(locale))
+                values.add(locale.toLangTag())
             }
+
+            Pair(names.toTypedArray(), values.toTypedArray())
+        }.subscribeK { (names, values) ->
+            lp.isEnabled = true
+            lp.entries = names
+            lp.entryValues = values
+            lp.summary = currentLocale.getDisplayName(currentLocale)
+        }
```

Comment:
These are just formatting collisions. Make sure you're using the new code style, not the obsolete one :) 

![image](https://user-images.githubusercontent.com/5982859/67160620-11530780-f353-11e9-870a-347a23c73a6f.png)